### PR TITLE
Create eip155-710420.json

### PR DESCRIPTION
**Pull request by nstamm10**

### CODE REVIEW

**Review by calmbeforeu — APPROVED** [review]

File: _data/chains/epi155-710420.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Tiltyard Subnet",
+  "chain": "TILTYARD",
+  "rpc": [
+    "https://subnets.avax.network/tiltyard/mainnet/rpc"
+  ],
```
Comment (connorch) [inline _data/chains/epi155-710420.json:6]: Looks like the CI failed due to formatting of this line.  I think it needs to be formatted like this?

```suggestion
  "rpc": ["https://subnets.avax.network/tiltyard/mainnet/rpc"],
```

**Review by ligi — CHANGES_REQUESTED** [review]
it states it quite clearly

**Comment by nstamm10** [comment]
@ligi Could you please explain? Unfortunately, that link is not working for me.

**Comment by ligi** [comment]
EPI!=EIP

**Comment by nstamm10** [comment]
Thanks!

**Pull request closed without merging**